### DISCUSSION TITLE
Does anyone still need to build MoltenVK using Xcode 11?

### DISCUSSION THREAD

**Discussion by billhollings** [KhronosGroup/MoltenVK#1850] [General]
MoltenVK currently supports building using Xcode as far back as Xcode 11.

As time goes by, it becomes more difficult to maintain functionality with this restriction.

Does anyone still need to build MoltenVK using Xcode 11 (published 2019)?

To keep this clean, please post an answer only if one or more of these apply:
- You still need support for Xcode 11.
- You up-to-recently needed support for Xcode 11, but no longer do.

**Comment by billhollings** [comment]
We have received no comments here yet. 

There are significant upcoming projects in MoltenVK, relying on newer features, that will benefit from not having to work around limitations in Xcode 11.

Please provide a comment soon, if one or more of these apply:
- You still need support for Xcode 11.
- You up-to-recently needed support for Xcode 11, but no longer do.

If no responses are received soon, we intend to remove support for Xcode 11 within the next one or two MoltenVK releases.

**Comment by cdavis5e** [comment]
@js6i @stefand @etang-cw Is this still needed to build CrossOver? As I recall, this was the original reason I requested this.

**Comment by mrpippy** [comment]
@cdavis5e No, we're building CrossOver with Xcode 13 now, we're fine with dropping support for 11.

**Comment by billhollings** [comment]
PR #1977 drops official support for using Xcode 11 to build MoltenVK.

`MVK_XCODE_12` guards have been left in place to allow devs to possibly continue to attempt to build existing MoltenVK code using Xcode 11, even though it's not officially supported. Such devs may need to add their own additional `MVK_XCODE_12` guards for any Xcode 12 API features added after this change.
